proxy/run.go

```diff
@@ -337,10 +341,17 @@ func listenAndServe(p *Proxy, mux *http.ServeMux, ctx context.Context, logger *z
 	return err
 }
 
-func resolveAndListen(address string) (net.Listener, error) {
-	tcpAddr, err := net.ResolveTCPAddr("tcp", address)
-	if err != nil {
-		return nil, err
+func resolveAndListen(address string, cert, key string) (net.Listener, error) {
```

Comment:
any particular reason why `cert` doesn't have the type but `address` has it?

Reply:
No, good catch.